src/SourceBuild/tarball/content/repos/known-good.proj

```diff
@@ -43,8 +43,8 @@
         <!-- Tier 2 -->
         <RepositoryReference Include="linker" />
         <RepositoryReference Include="msbuild" />
-        <!-- <RepositoryReference Include="runtime-portable" /> -->
-        <!-- <RepositoryReference Include="runtime" /> -->
+        <RepositoryReference Include="runtime-portable" />
```

Comment:
There are some disabled runtime project references that should get re-enabled with this PR.  e.g. deployment-tools, vstest, msbuild.